### DISCUSSION TITLE
45 LED Cake

### DISCUSSION THREAD

**Discussion by CrisisFilms** [gpstar81/GPStar-proton-pack#328] [Ideas]
Looking to add support for a 45 LED ring (https://www.amazon.com/gp/product/B08GPCB7K3) for the inner cyclotron.

**Comment by nomakewan** [comment]
It will not be officially supported because it would require that a user reduce their other LEDs. That said, you can follow the same instructions I provided in your other discussion thread to do it in a custom firmware (just change 40 to 45): #260
